exercises/complex-number/src/lib.rs

```diff
@@ -0,0 +1,70 @@
+#[derive(Clone, Copy, Debug, PartialEq)]
+pub struct Complex {
+    // Add in the fields
+}
+
+impl Complex {
+    pub fn new(_re: f64, _im: f64) -> Complex {
+        unimplemented!()
+    }
+
+    pub fn exp(&self) -> Complex {
```

Comment:
I wouldn't know what needs to be implemented in this operation. I see `exp` and think exponentiation, but there's no argument passed in, and that's not what the example does.

Please add a short comment to this function describing what operation should be performed.

Reply:
I decided to use `Complex::exp` for `e^(a + bi)` to be consistent with what the [standard library](file:///home/shingtaklam1324/.rustup/toolchains/nightly-x86_64-unknown-linux-gnu/share/doc/rust/html/std/primitive.f64.html#method.exp) does there. It seems to be more common to use `exp` for `e^x` and `pow(i / f / c)` for `a^b` within Rust's ecosystem